packages/components/src/tooltip/index.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useCallback, useRef, useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import { createPopper } from '@popperjs/core';
+
+import { arrayToHash } from 'nebenan-helpers/lib/data';
+import { getPopperOptions } from './utils';
+import styles from './index.module.scss';
+
+const knownTypes = arrayToHash(['left', 'right', 'top', 'bottom']);
+
+const Tooltip = (props) => {
+  const [isOpen, setOpen] = useState(false);
+  const { type, text, children, ...cleanProps } = props;
+  const selectedType = knownTypes[type] ? type : 'top';
+  const className = clsx(styles.tooltip, props.className, {
+    [styles.isActive]: isOpen,
+  });
+
+  const handleOpen = useCallback((event) => {
+    event.stopPropagation();
+    setOpen(true);
+  }, [isOpen]);
+
+  const handleClose = useCallback((event) => {
+    event.stopPropagation();
+    setOpen(false);
+  }, [isOpen]);
+
+  const refElement = useRef(null);
+  const refTooltip = useRef(null);
+  const refArrow = useRef(null);
+
+  useEffect(() => {
```

Comment:
I think `useLayoutEffect` is needed here, please check the documentation

https://github.com/popperjs/react-popper/blob/master/src/usePopper.js